### PR TITLE
Importing SetOfTomograms as default

### PR DESCRIPTION
**Pull request by DavidHerreros**
- Changes added to `protocol_import_tomograms` allowing to import a SetOfTomograms of size 1 (importing a Tomogram object alone is no longer available).

- Changes added to `protocol_import_coordinates` to work with SetOfTomograms and create outputs that can be understood by `protocol_tomo_extraction` from Eman plugin.

This changes are required by [scipion-em/scipion-em-eman2/pull/34](https://github.com/scipion-em/scipion-em-eman2/pull/34) to work properly.

### CODE REVIEW

File: tomo/protocols.conf [hunk 1]
```diff
@@ -17,6 +17,8 @@ Tomography = [
 	{"tag": "protocol", "value": "ProtTsEstimateCTF",      "text": "default"}
 	]},
 	{"tag": "section", "text": "Reconstruction", "children": []},
+	{"tag": "section", "text": "Particles", "children": []},
```
Comment (adrianq) [inline tomo/protocols.conf]: What is the purpose of these two lines @EstrellaFG ?

File: tomo/protocols/protocol_import_coordinates.py [hunk 2]
```diff
@@ -29,9 +29,12 @@
 
 import pyworkflow.protocol.params as params
 from pyworkflow.utils import importFromPlugin
+import os
```
Comment (adrianq) [inline tomo/protocols/protocol_import_coordinates.py]: Imports should be made from more general to more specific.
So, first library ones such as os, sys, numpy, etc
Then pyworkflow and familiy...
And finally tomo ones.

File: tomo/protocols/protocol_import_tomograms.py [hunk 3]
```diff
@@ -117,10 +117,11 @@ def importTomogramsStep(
                     tomo.setAcquisition(self._extractAcquisitionParameters(fileName))
                     tomoSet.append(tomo)
 
-        if tomoSet.getSize() > 1:
-            self._defineOutputs(outputTomograms=tomoSet)
-        else:
-            self._defineOutputs(outputTomogram=tomo)
+        ### Change needed to import always a set
```
Comment (adrianq) [inline tomo/protocols/protocol_import_tomograms.py]: Just remove the commented out code if we are not actually using it and there is no plan to use it in the short term

File: tomo/protocols/protocol_import_coordinates.py [hunk 4]
```diff
@@ -72,22 +75,29 @@ def _insertAllSteps(self):
...
-            if importTomogram is not None:
-                def addCoordinate(coord):
-                    coord.setVolume(importTomogram)
-                    coordsSet.append(coord)
-
-                # Parse the coordinates in the given format for this micrograph
-                ci.importCoordinates3D(coordFile, addCoordinate)
+        for tomo in importTomograms.iterItems():
+            tomoName = os.path.splitext(tomo.getFileName())[0]
+            tomoName = basename(tomoName)
```
Comment (adrianq) [inline tomo/protocols/protocol_import_coordinates.py]: I would merge these two lines in a single line or at least use different variable names

File: tomo/protocols/protocol_import_coordinates.py [hunk 5]
```diff
@@ -72,22 +75,29 @@ def _insertAllSteps(self):
...
+            for coordFile, fileId in self.iterFiles():
+                fileName = os.path.splitext(coordFile)[0]
+                fileName = "import_" + basename(fileName)
```
Comment (adrianq) [inline tomo/protocols/protocol_import_coordinates.py]: I would merge these two lines in a single line or at least use different variable names

**Comment by EstrellaFG** [comment]
> Everything seems to work correctly, except:
> 
> * summary of protocol import tomograms
> * tests (if I run scipion test tomo.tests.test_tomo_base some of them fail)

The tests that are now failing are on the tilt series side, so I guess they will be fixed in https://github.com/scipion-em/scipion-em-tomo/pull/17

**Pull request merged**